tests/imfile-wildcards-dirs-multi2.sh

```diff
@@ -28,19 +28,17 @@ do
 	do
 		echo "Make rsyslog.input.dir$i/dir$j/testdir"
 		mkdir rsyslog.input.dir$i/dir$j
-		./msleep 25
 		mkdir rsyslog.input.dir$i/dir$j/testdir
-		./msleep 25
 		./inputfilegen -m 1 > rsyslog.input.dir$i/dir$j/testdir/file.logfile
-		./msleep 50
 	done
 	ls -d rsyslog.input.*
 
+	./msleep 250
```

Comment:
why do we need this?

Reply:
Yes imfile needs a little time to read and process the file at least. Otherwise we would generate same files again without any pause.